client/my-sites/plan-features/header.jsx

```diff
@@ -31,13 +31,19 @@ import isSiteAutomatedTransfer from 'calypso/state/selectors/is-site-automated-t
 import { getCurrentPlan } from 'calypso/state/sites/plans/selectors';
 import { getSiteSlug } from 'calypso/state/sites/selectors';
 import { getSelectedSiteId } from 'calypso/state/ui/selectors';
+import { isMarketplaceFlow } from '../plugins/flows';
```

Comment:
Dusty pointed out on my last PR is recommended to import using the absolute path. It will also ask to change the order of the import.

```suggestion
import { isMarketplaceFlow } from 'calypso/my-sites/plugins/flows';
```